server/firebaseConfig.ts

```diff
@@ -1,14 +1,15 @@
-import { initializeApp } from 'firebase/app';
+import { FirebaseOptions, initializeApp } from 'firebase/app';
 import { getAuth } from 'firebase/auth';
+import { Config } from './config';
 
-const firebaseConfig = {
-  apiKey: process.env.API_KEY,
-  authDomain: process.env.AUTH_DOMAIN,
-  projectId: process.env.PROJECT_ID,
-  storageBucket: process.env.STORAGE_BUCKET,
-  messagingSenderId: process.env.MESSAGING_SENDER_ID,
-  appId: process.env.APP_ID,
-  measurementId: process.env.MEASUREMENT_ID,
+const firebaseConfig: FirebaseOptions = {
+  apiKey: Config.firebase.apiKey,
+  authDomain: Config.firebase.authDomain,
+  projectId: Config.firebase.projectId,
+  storageBucket: Config.firebase.storageBucket,
+  messagingSenderId: Config.firebase.messagingSenderId,
+  appId: Config.firebase.appId,
+  measurementId: Config.firebase.measurementId,
 };
 
 const app = initializeApp(firebaseConfig);
```

Comment:
can probably just consolidate into `initializeApp(Config.firebase)`

Reply:
And potentially consider just moving into `config/index.ts` or its own file under `config`